tests/api_fastapi/core_api/routes/ui/test_structure.py

```diff
@@ -51,8 +52,15 @@ def make_dag(dag_maker, session, time_machine):
         serialized=True,
         session=session,
         start_date=pendulum.DateTime(2023, 2, 1, 0, 0, 0, tzinfo=pendulum.UTC),
+        schedule=(
+            Asset(uri="s3://bucket/next-run-asset/1", name="asset1")
+            & Asset(uri="s3://bucket/next-run-asset/2", name="asset2")
+            & AssetAlias("example-alias")
+        ),
```

Comment:
I am having trouble testing the `Trigger`. I seems to work when I use the application, but in test, the dependency seems to not be resolved.

Reply:
Fixed.